SuperShop.Runner/Program.cs

```diff
@@ -1,14 +1,43 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Globalization;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace SuperShop.Runner
 {
-    class Program
+    public class Program
     {
         static void Main(string[] args)
+        {
+            new Disposable();
+        }
+
+        public static async void Test()
```

Comment:
* 'public' members must come before 'private' members
 * Change method name to TestAsync